scan-car/src/components/navigation.js

```diff
@@ -0,0 +1,19 @@
+import './navigation.css';
+import logo from './logo.png';
+//import logo from '.\.\public\logo.png';
+
+function Navigation () {
+    return (
+        <header>
+            <div class="navtabs">
+                <div class ="navtab-Logo"><img src={logo} alt="ScanCar Logo" height="40px"></img></div>
+                <div class="navTabActive">Booking Management</div>
+                <div class ="navtab">Rental Management</div>
+                <div class="navtab">Group Management</div>
+            </div>
+            
+        </header>
+    )
+}
```

Comment:
The indentation and formatting is a little messy here. Make it a habit to use Prettier.

Also, using semantic HTML would be much better here (Look under _HTML <nav> element_ here [https://www.w3schools.com/html/html5_semantic_elements.asp] for ref.